fusion-plugin-i18n-react/src/__tests__/use-translations.test.browser.js

```diff
@@ -0,0 +1,30 @@
+// @flow
```

Comment:
Purpose of tests in this file (no static analysis happens in these unit tests):
- `useTranslations` hook gets `translate` function from `I18nContext.Provider`
- `translate` is called properly